_targets_subset_benchmarking.R

```diff
@@ -0,0 +1,102 @@
+# Load packages required to define the pipeline:
+library(targets)
+library(lubridate)
+library(purrr, quietly = TRUE)
+
+# Set target options:
+tar_option_set(
+  workspace_on_error = TRUE,
+  packages = c("wweval"),
+  memory = "transient",
+  garbage_collection = TRUE,
+  format = "rds", # default storage format
+  error = "continue" # tells errored targets to return NULL rather than
+  # have whole pipeline fail
+)
+
+setup_interactive_dev_run <- function() {
+  list.files(file.path("wweval", "R"), full.names = TRUE) |>
+    purrr::walk(source)
+  tar_option_set(
+    packages = c(
+      "cmdstanr",
+      "rlang",
+      "tibble",
+      "ggplot2",
+      "dplyr",
+      "lubridate",
+      "cmdstanr",
+      "tidybayes",
+      "cfaforecastrenewalww",
```

Comment:
This presumably shouldn't be here?

Reply:
To avoid silent deps you might want to uninstall `cfaforecastrenewalww` locally.